src/scorer.py

```diff
@@ -4,19 +4,19 @@
 import re
 from typing import Tuple, List, Optional
 from collections import Counter
-
-
-from .objdump import objdump, get_arch
+from .objdump import ArchSettings, objdump, get_arch
 
 
 @dataclass(init=False, unsafe_hash=True)
 class DiffAsmLine:
     line: str = field(compare=False)
     mnemonic: str
+    has_symbol: bool
```

Comment:
given the creation of Line in objdump.py and only mnemonics being passed to SequenceMatcher, is DiffAsmLine still needed?

Reply:
hmmm! yea we can definitely consolidate here